server/api.go

```diff
@@ -45,12 +50,58 @@ type PRDetails struct {
 	Reviews            []*github.PullRequestReview `json:"reviews"`
 }
 
+type HTTPHandlerFuncWithUser func(w http.ResponseWriter, r *http.Request, userID string) http.HandlerFunc
```

Comment:
Why does this need to return a `http.HandlerFunc`? Is ` func(w http.ResponseWriter, r *http.Request, userID string)` as signature enough?